<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icons/images/calculator.png</normaloff>:/icons/images/calculator.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>270</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>270</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>270</width>
      <height>340</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>270</width>
      <height>340</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>270</width>
      <height>340</height>
     </size>
    </property>
    <widget class="QPushButton" name="pushButtonClear">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>C</string>
     </property>
     <property name="shortcut">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPercentage">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="shortcut">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMultiply">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="shortcut">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDecrease">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="shortcut">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="shortcut">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="shortcut">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="shortcut">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonIncrease">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="shortcut">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="shortcut">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="shortcut">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="shortcut">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="shortcut">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="shortcut">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="shortcut">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAnswer">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>=</string>
     </property>
     <property name="shortcut">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDecimal">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="shortcut">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton0">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>131</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="shortcut">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPlusMinus">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
     <property name="shortcut">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDivide">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>/</string>
     </property>
     <property name="shortcut">
      <string>+</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
